src/Preprocessor.zig

```diff
@@ -966,9 +973,6 @@ fn expandMacroExhaustive(pp: *Preprocessor, tokenizer: *Tokenizer, buf: *ExpandB
 
                     var res = try pp.expandFuncMacro(macro_tok.loc, macro, &args, &expanded_args);
                     defer res.deinit();
-                    for (expanded_args.items) |arg| {
```

Comment:
I'm unsure about this change - was this separate from the `expanded_args.deinit()` for a reason?

Reply:
I have written that code but I have no idea why I did this, your change looks good to me.